### DISCUSSION TITLE
Is it possible to decrease amount of wallets?

### DISCUSSION THREAD

**Discussion by Opty1712** [WalletConnect/walletconnect-monorepo#3581] [Developer Support] [answered]
Hi, I want to use only MetaMask, CoinBase, TrustWallet and Ledger. Is it possible?
I found `mobileWallets / desktopWallets` as props to `<Web3Modal ... />` but there are no examples, so I failed

**Comment by MuckT** [comment, marked as answer]
I'd argue you might not actually want to limit the available wallets to only certain wallets as your effectively reducing your number of possible dapps users, but if you wanted to the code below should do it. If you just want to control the preferred wallets simply remove the `explorerExcludedWalletIds="ALL"` prop.

```JavaScript
<Web3Modal
  projectId={'DAPP_PROJECT_ID'}
  ethereumClient={'DAPP_ETHEREUM_CLIENT'}
  explorerExcludedWalletIds="ALL"
  explorerRecommendedWalletIds={[
    // MetaMask ID https://walletconnect.com/explorer/metamask
    'c57ca95b47569778a828d19178114f4db188b89b763c899ba0be274e97267d96',
    // CoinBase ID https://walletconnect.com/explorer/coinbase-wallet
    'fd20dc426fb37566d803205b19bbc1d4096b248ac04548e3cfb6b3a38bd033aa',
    // Trust Wallet ID https://walletconnect.com/explorer/trust-wallet
    '4622a2b2d6af1c9844944291e5e7351a6aa24cd7b23099efac1b2fd875da31a0',
    // Ledger Live ID https://walletconnect.com/explorer/ledger-live
    '19177a98252e07ddfc9af2083ba8e07ef627cb6103467ffebb3f8f4205fd7927'
   ]}
/>
```

**Reply by Opty1712** [reply]
Thank you, it works!
<img width="372" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/14236476/059c4ed4-83aa-48e0-99fe-193973f88f26">
